scripts/package-appimage.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -e
 
+export APPIMAGE_EXTRACT_AND_RUN=1
```

Comment:
Would it be a good idea to avoid exporting this, and just prepending it where it's needed?
Just to avoid unexpected behavior.

Reply:
>Would it be a good idea to avoid exporting this, and just prepending it where it's needed?
Just to avoid unexpected behavior.

It is suggested to instead use the env variable instead of passing the `--appimage-extract-and-run` flag, because if at some point some of the deploy plugins get used they will break since you can't pass the flag when calling the plugins. 

All the variable does is make sure that any appimage launched in the CI will still work even if fuse is missing/broken, that's the only behaviour it changes, but let me know what you think after what I said.

Reply:
> > Would it be a good idea to avoid exporting this, and just prepending it where it's needed?
> > Just to avoid unexpected behavior.
> 
> It is suggested to instead use the env variable instead of passing the `--appimage-extract-and-run` flag, because if at some point some of the deploy plugins get used they will break since you can't pass the flag when calling the plugins.
> 
> All the variable does is make sure that any appimage launched in the CI will still work even if fuse is missing/broken, that's the only behaviour it changes, but let me know what you think after what I said.

We know what the environment variable means. The suggestion is that instead of exporting it, you can do it like this:
```
APPIMAGE_EXTRACT_AND_RUN=1 ./appimagetool blabla
```
Which will limit the environment to only appimagetool, and makes it clear what the environment variable is for.

Reply:
>We know what the environment variable means. The suggestion is that instead of exporting it, you can do it like this:

Alright that way it will also be passed down to everything being called by appimagetool and there wouldn't be issues if the plugins ever get used. 

However this means that the variable has to be passed to every appimage being called in the CI, I'm not a fan of doing that but I assume this is the only time that gets done here anyway.

Reply:
This is the only appimage that is being called, and this will not propagate to every other steps in the CI because it's run inside it's own shell, not sourced. If we needed this, its better to add it to CI configuration rather than this script.